static/docs/user-guide/howto/keep-data-and-cache-on-external-drive.md

```diff
@@ -0,0 +1,200 @@
+external drive to the internal one. For example let's say that the size of the
+external drive is 16TB, while the local drive is only 320GB.
+
+In this case we would like to process the data where it is already located (on
+the external drive). We also would like to store the results there, as well as
+the <abbr>cached</abbr> files.
+
+<details>
+
+### A simple approach: initialize the workspace on the external drive itself
+
+The easiest way to do this would be to initialize the workspace on the external
+drive itself. If we assume that the external drive is mounted on
+`/mnt/external-drive/` and the data is on `/mnt/external-drive/data/raw`, then
+it could be done like this:
+
+```dvc
+$ cd /mnt/external-drive/
+$ git init
+$ dvc init
+$ dvc add data/raw
+...
+```
+
+But often this is not possible (or preferable). Another solution is described in
+the rest of this page.
+
+</details>
+
+Most likely, the best approach for this is to setup the <abbr>workspace</abbr>
+in our local drive, while all the data files and their caches stay on the
+external drive.
+
+## Make the data directory accessible
+
+Let's assume that the external drive is mounted on `/mnt/external-drive/`. First
+we have to make sure that we can read and write this data directory:
+
+```dvc
+$ sudo chown <username>: -R /mnt/external-drive/
+$ chmod u+rw -R /mnt/external-drive/
+```
+
+> Or refer to
+> [User Account Control](https://docs.microsoft.com/en-us/windows/security/identity-protection/user-account-control/user-account-control-overview)
+> for Windows.
+
+## Create a DVC project with external cache
+
+An [external cache](/doc/user-guide/external-outputs) resides outside of the
+workspace directory. Let's create a directory for it on `/mnt/external-drive/`:
+
+```dvc
+$ mkdir -p /mnt/external-drive/dvc-cache
+```
+
+Now you can initialize a <abbr>project</abbr> on your home directory:
+
+```dvc
+$ cd ~/project/
+$ git init
+$ dvc init
+```
+
+> If this was an existing project, we could preserve the content of its cache by
+> moving it to the external directory with
+> `mv -a .dvc/cache/* /mnt/external-drive/dvc-cache/`
+
+Configure it to use the external directory as <abbr>cache</abbr>:
+
+```dvc
+$ dvc config cache.dir /mnt/external-drive/dvc-cache
+$ cat .dvc/config
+[cache]
+dir = /mnt/external-drive/dvc-cache
+```
+
+Commit changes to git:
+
+```dvc
+$ git add .dvc/config
+$ git commit -m 'Initialize DVC with external cache'
+```
+
+## Tracking external dependencies and outputs
+
+Now, when you refer to the data files and directories in DVC commands, you'll
+have to **use their absolute path**. <abbr>DVC-files</abbr> will be created in
+the <abbr>project</abbr>, and you can track their modifications with Git as
+usual.
+
+For example, let's say that the raw data files are in
+`/mnt/external-drive/data/raw/`, and you are cleaning them up. You could do it
+like this:
+
+```dvc
+$ dvc add /mnt/external-drive/data/raw
+...
+$ dvc run \
```

Comment:
if you do this, it's better to have a separate directory per project on the external drive. Or, again, I don't quite understand the motivation and use case behind this configuration.

Reply:
> if you do this, it's better to have a separate directory per project on the external drive

I don't understand your request here. It is just an example, why should we talk about several projects?

> I don't quite understand the motivation and use case behind this configuration.

I have tried to explain it here: https://github.com/iterative/dvc.org/pull/732#discussion_r339773638

I will try to revise the motivation including this one too:

> Recently I have seen another discussion where a user wants to keep all the data and cache on an external directory, on a fixed absolute path, so that he can search easily the code of the project.